From 5fd51351615e781e5d696087eeb01f30fe13fe9a Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Tommi=20My=C3=B6h=C3=A4nen?= <ext-tommi.1.myohanen@nokia.com>
Date: Fri, 27 Nov 2009 12:19:12 +0200
Subject: [PATCH] camerabin: don't store incorrect pre-nightmode framerate

Before switching to night mode gst_camerabin_handle_scene_mode()
stores the current framerate. There is no need to store it again
in camerabin_setup_src_elements() because it may store wrong
framerate (when camerabin is started in night mode).
---
 gst/camerabin/gstcamerabin.c |    2 --
 1 files changed, 0 insertions(+), 2 deletions(-)

diff --git a/gst/camerabin/gstcamerabin.c b/gst/camerabin/gstcamerabin.c
index f5bbe22..bf17bcc 100644
--- a/gst/camerabin/gstcamerabin.c
+++ b/gst/camerabin/gstcamerabin.c
@@ -525,8 +525,6 @@ camerabin_setup_src_elements (GstCameraBin * camera)
   if (camera->fps_n > 0 && camera->fps_d > 0) {
     if (camera->night_mode) {
       GST_INFO_OBJECT (camera, "night mode, lowest allowed fps will be forced");
-      camera->pre_night_fps_n = camera->fps_n;
-      camera->pre_night_fps_d = camera->fps_d;
       detect_framerate = TRUE;
     } else {
       gst_structure_set (st,
